tests/operators/test_bash.py

```diff
@@ -123,6 +125,41 @@ def test_command_not_found(self):
         ):
             BashOperator(task_id='abc', bash_command='set -e; something-that-isnt-on-path').execute({})
 
+    def test_command_without_cwd(self):
+        val = "xxxx"
+        op = BashOperator(task_id='abc', bash_command=f'set -e; echo "{val}";')
+        line = op.execute({})
+        assert line == val
+
+    def test_command_with_cwd(self):
+
+        test_cwd_folder = "/tmp/airflow/test_bash/test_command_with_cwd"
+        test_file_name = "py_test.txt"
+        test_cwd_file_path = f"{test_cwd_folder}/{test_file_name}"
+        # The command is echo a string to a file into cwd folder and return the string as the output
+        test_cmd = f'set -e; echo "xxxx" |tee {test_file_name}'
+
+        # Test the the bash_cwd doesn't exist
+        if os.path.exists(test_cwd_folder):
+            _shutil.rmtree(test_cwd_folder)
```

Comment:
It’d be easier to use `tempfile.TemporaryDirectory` instead of manually cleaning things up. This would also be friendly for cross-platform support (we are working toward having a Windows-compatible task runner).

Reply:
I have a question that what I want to do is make sure the directory doesn't exist at this time, so I try to clean it.
I check the source codes of `TemporaryDirectory`, the `__exit__` method is `_shutil.rmtree(name)`.
But I agree the following test we can use `TemporaryDirectory` to do this

Reply:
Changed it.